fsspec/tests/test_mapping.py

```diff
@@ -44,7 +44,7 @@ def test_getitems_errors(tmpdir):
 
 
 def test_ops():
-    MemoryFileSystem.store.clear()
```

Comment:
Hm, I wonder if anyone is using this outside of tests 🤔 Worried about breaking backward compatibility for someone. An alternative might be to create `memfs2` or `LocalMemoryFileSystem` or something like that instead of altering `MemoryFileSystem` with this change. Could also keep it as it was before and overwrite `self.store` in `__init__`  if `global_store=False`